.travis.yml

```diff
@@ -2,6 +2,7 @@ sudo: false
 language: ruby
 cache: bundler
 rvm:
-  - 2.2.7
-  - 2.3.4
-  - 2.4.1
+  - 2.4.10
+  - 2.5.8
+  - 2.6.6
+  - 2.7.1
```

Comment:
https://www.ruby-lang.org/en/downloads/releases/


Release Version | Release Date
-- | -- 
Ruby 2.7.1 | 2020-03-31
Ruby 2.6.6 | 2020-03-31
Ruby 2.5.8 | 2020-03-31
Ruby 2.4.10 | 2020-03-31

Ruby 2.3 hasn't had a release since 2018